### ISSUE TITLE
Widen search for missing transcripts

### ISSUE THREAD

**Issue by pattonwebz** [equalizedigital/accessibility-checker#1545]
## Is your feature request related to a problem? Please describe.

A customer reported issues finding transcripts that were visually beside the vidoes. On looking I seen that they were visually beside but the builder they has containers and wrappers that put them more than 3 dom traversal levels away from the frame.

## Describe the solution you'd like

The rule only looks up 3 levels, a solution is to widen it to 4 levels of traversal. There search of more levels upwards does reduce the performance of the rule by a tiny amount but it isn't really a measurable impact on pages that only have a small number of videos to find for.
